assets/charts/components/azure-arc-onboarding/templates/rbac.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: zz-azure-arc-onboarding
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: true
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+  # Allow core volume types.
+  volumes:
+  - 'configMap'
+  - 'emptyDir'
+  - 'projected'
+  - 'secret'
+  - 'downwardAPI'
+  # Assume that persistentVolumes set up by the cluster admin are safe to use.
+  - 'persistentVolumeClaim'
+    # This capability is required for kube-aad-proxy deployment from azure-arc
+    # to mount hostPath volumes.
+  - 'hostPath'
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'RunAsAny'
+  fsGroup:
+    rule: 'RunAsAny'
+  readOnlyRootFilesystem: false
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: azure-arc-onboarding-for-psp
+    #  annotations:
+    #    "helm.sh/hook": "pre-install,pre-delete"
+rules:
+- apiGroups: ['policy']
+  resources: ['podsecuritypolicies']
+  verbs:     ['use']
+  resourceNames:
+  - zz-azure-arc-onboarding
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: azure-arc-onboarding
+    #  annotations:
+    #    "helm.sh/hook": "pre-install,pre-delete"
+subjects:
+- kind: ServiceAccount
+  name: azure-arc-onboarding
+  namespace: {{ .Values.namespace }}
```

Comment:
Why not `.Release.Namespace` ? Then we don't need `namespace` field in values and in struct.